.gitignore

```diff
@@ -8,3 +8,7 @@ spec/dummy/log/*.log
 spec/dummy/public/assets
 spec/dummy/tmp/
 spec/dummy/.sass-cache
+
+# bower-rails
```

Comment:
It seems like configuration to me. I'm not sure we should/need to re-generate it every time.